src/integration/cd/go/framework/ldap/LdapIntegrationTest.java

```diff
@@ -107,4 +118,60 @@ public void shouldSearchUsersFromMultipleSearchBases() throws Exception {
         assertThat(users, containsInAnyOrder(new User("pbanks", "P.Banks", "pbanks@example.com"), new User("sbanks", "S.Banks", "sbanks@example.com")));
         verify(ldapSpy).closeContextSilently(any(DirContext.class));
     }
+
+
+    @Test
+    public void search_shouldLimitTheSearchResults() throws Exception {
+        final LdapConfiguration ldapConfiguration = ldapConfiguration(new String[]{"ou=Employees,ou=Enterprise,ou=Principal,ou=system", "ou=Clients,ou=Enterprise,ou=Principal,ou=system"});
```

Comment:
Since the next test tests across search bases, maybe limit this to one search base?